### ISSUE TITLE
Yabai -m space --focus - Not Switching Spaces

### ISSUE THREAD

**Issue by BrianAre** [koekeishiya/yabai#2138]
first time user. I turned off SIP, gave yabai accessibility/screenrecording permission, but my skhd shortcuts to switch between spaces is not working. Everything else works great.

The yabai -V output is reading the event from my skhd shortcut, but nothing happens on screen. It doesn't switch to the next space. It might be an error on my end, but it could be a bug.

On M2 MBA, Sonoma 14.2.1

(on left yabai -V; on right, skhd script
<img width="1579" alt="Screenshot 2024-03-03 at 1 05 06 AM" src="https://github.com/koekeishiya/yabai/assets/137826006/62b6db9e-b7e2-460b-a4a2-21c12bfb4280">

**Comment by BrianAre** [comment]
Resolved this by troubleshooting the scripting addition

I had to run this command:
sudo nvram boot-args=-arm64e_preview_abi

**Comment by IanQS** [comment]
@BrianAre did you run that command alone and everything started working? Or were there other things you had to do, too? What context did you run that in, do you remember?

**Comment by BrianAre** [comment]
> @BrianAre did you run that command alone and everything started working? Or were there other things you had to do, too? What context did you run that in, do you remember?

I tried multiple things before I ran this command. Once I ran this command everything just worked. Here's what I did before:

1. Disabled SIP
2. Made sure it had 'accessibly' and 'screen recording' permission
3. Installed Dev. Version
4. Configured Scripting Addition (I did it once before, but had numerous typos - corrected it)
5. Then it still didn't work until I saw an old post (https://github.com/koekeishiya/yabai/issues/1054) that said to  run: sudo nvram boot-args=-arm64e_preview_abi
6. After that, it worked!

I'm not sure step 3 helped; i'm sure it can work with the current release and not the Dev Version.
